gitops/prer/create_gitops_prs.go

```diff
@@ -71,6 +73,7 @@ var (
 	gitopsRuleName         SliceFlags
 	gitopsRuleAttr         SliceFlags
 	dryRun                 = flag.Bool("dry_run", false, "Do not create PRs, just print what would be done")
+	stamp                  = flag.Bool("stamp", false, "Stamp results of gitops targets with volatile information")
```

Comment:
```suggestion
	stamp                  = flag.Bool("stamp", false, "Stamp results of gitops targets with volatile information")
	dryRun                 = flag.Bool("dry_run", false, "Do not create PRs, just print what would be done")
```
Historically we kept `dryRun` as a last argument definition.